src/index.html

```diff
@@ -5,10 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#" class="header_logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="header_nav">
+          <ul class="nav_list">
+            <li class="nav__item">
+              <a href="#" class="nav_link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#" class="nav_link is-active">Apple</a>
            </li>
            
            <li class="nav__item">
```

Add empty line between multiple sibling blocks of html

Reply:
I added an empty line between multiple sibling blocks.